wa/workloads/gfxbench/uiauto/app/src/main/java/com/arm/wa/uiauto/gfxbench/UiAutomation.java

```diff
@@ -122,7 +128,7 @@ public void clearFirstRun() throws Exception {
         }
 
         UiObject home =
-            mDevice.findObject(new UiSelector().resourceId(packageID + "main_homeBack")
+            mDevice.findObject(new UiSelector().resourceId(packageID + "main_view_back")
```

Comment:
Is this version dependent? And if so, should we be checking for both?

Reply:
I don't think so no. This appears to be a background element that is present only the homescreen which has been updated to now be a generic background element of the application which is shared by any of the main screens which is used to detect when any popups have finished closing.